shoov/modules/custom/shoov_restful/plugins/restful/node/ci_build_status/1.0/ShoovCiBuildStatusResource.class.php

```diff
@@ -70,17 +70,27 @@ protected function isValidEntity($op, $entity_id) {
     $params = array(
       '@id' => $entity_id,
     );
+    $node = node_load($entity_id);
+    $user = $this->getAccount();
+    if (!$entity_id) {
```

Comment:
this should move to the `controllerInfo()` - simply remove the part that allows passing an empty url.